content/logs/explorer/_index.md

```diff
@@ -107,6 +107,17 @@ Follow the [log graphing guide][5] to learn more about all the graphing options.
 [3]: /logs/processing
 [5]: /logs/explorer/analytics
 
+{{% /tab %}}
+{{% tab "Log Patterns" %}}
+
+Investigating into large volume of logs is really time consuming, you can spend hours in it and still understand a fraction of it. Quite luckily, applicative logs are repetitive and pattern can be detected to automatically bring structure to the problem and help you to quickly see what matters or exclude what's pollute the general understanding.
```

Comment:
first sentence seems off.  maybe "Investigating into large volume of logs is really time consuming" -> "Investigating large volumes of log data can be time consuming"

No need for "really" or "quite luckily"

"applicative" isn't a word.  You could say "Since application logs tends to follow patterns, Datadog is able to detect them automatically to bring structure to your data"

Reply:
I'd avoid "pollute" here as well.